lib/active_admin/exception_app.rb

```diff
@@ -0,0 +1,79 @@
+require 'active_admin/view_helpers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.